project.godot

```diff
@@ -26,6 +26,10 @@ window/size/viewport_width=1280
 window/size/viewport_height=720
 window/stretch/mode="canvas_items"
 
+[gui]
+
+theme/custom_font="res://assets/fonts/VCR_OSD_MONO_1.001.ttf"
```

Comment:
I tried this before and it made lots of stuff look like poopoo so I reverted it - how do you think the UI looked with this forced on everything?

Reply:
<img width="524" alt="image" src="https://github.com/user-attachments/assets/28ada29b-64ba-4127-b75f-969b5e5de76f">
It looks fine tbh. We can always add back the default godot sans-serif font for text if we need to.

Reply:
Sure. I wanted things to line up pixel perfect but honestly it's not really needed, the font looks pretty good. I'm wondering if we should get rid of the bold though, that looks a bit off?

Also I'm wondering if instead of listing the roles and who did that role, we instead list each person and below list what roles they did. Might be more organized and allow us to put in funny stuff

Reply:
Yeah I agree the bold looks bad. I might change it to just be a size increase.